### PR TITLE
style(ctb): clean up legacy comments and errs

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Cleans up comments and errors for legacy contracts. It's fine to update
the bytecode for these legacy contracts since we're really only using
them for their interfaces.

### CODE REVIEW

File: packages/contracts-bedrock/contracts/legacy/AddressResolver.sol [hunk 1]
```diff
@@ -1,40 +0,0 @@
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.9;
-
-/* Library Imports */
-import { AddressManager } from "./AddressManager.sol";
-
-/**
- * @title AddressResolver
- */
-abstract contract AddressResolver {
```
Comment (maurelian) [inline packages/contracts-bedrock/contracts/legacy/AddressResolver.sol:10]: When did this cease to be in use?

**Review by maurelian — APPROVED** [review]

File: packages/contracts-bedrock/contracts/legacy/AddressResolver.sol (hunk 1 above)
Comment (smartcontracts) [reply packages/contracts-bedrock/contracts/legacy/AddressResolver.sol:10]: It was never in use for contracts-bedrock, not sure why it was copied in the first place

**Review by tynes — CHANGES_REQUESTED** [review]
This needs the bindings to be regenerated

**Comment by tynes** [comment]
Just pushed new bindings

**Review by tynes — APPROVED** [review]

**Review by tynes — APPROVED** [review]

**Pull request merged**